Flask_app/services/tasks.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any
+from ..models.models import Task
+from .. import db
+
+def create_task(model: Any, title: str, description: str):
+    task = model(
+        title = title,
+        description = description
+    )
+    db.session.add(task)
+    db.session.commit()
+
+    return task
+
+def get_all_tasks():
+    return Task.query.all()
+
+
+
+def get_task_by_id(id):
+    task = Task.query.get(id)
+    return task
+
+
```

Comment:
Use some linter for better code and formatting

https://code.visualstudio.com/docs/python/linting